chromebook-setup.sh

```diff
@@ -687,6 +687,18 @@ cmd_deploy_kernel_only()
     cmd_eject_storage
 }
 
+if grep -qi fedora /etc/os-release; then
+  os="fedora"
+else
+  os="debian"
+fi
+
+if [ "$os" = "fedora" ]; then
+  pkg_mgr="dnf"
+else
+  pkg_mgr="apt"
+fi
+
```

Comment:
This can be done with a single `if else` statement, and you don't need the os variable at all.